.github/workflows/deploy-prod.yml

```
@@ -50,6 +70,10 @@ jobs:
 
   deploy-core:
     if: ${{ inputs.deploy-core }}
+    env:
```

actionlint (1.7.8)

when a reusable workflow is called with "uses", "env" is not available. only following keys are allowed: "name", "uses", "with", "secrets", "needs", "if", and "permissions" in job "deploy-core"

(syntax-check)